src/convertToObject.js

```diff
@@ -1,12 +1,22 @@
-'use strict';
-
-/**
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const filteredArr = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item);
+
+  const newArr = filteredArr
+    .map((item) => item.split(':'))
+    .map((item) => item.map((subItem) => subItem.trim()));
+
+  const resultObject = newArr.reduce((obj, item) => {
```

Comment:
The variable name 'resultObject' is generic. Consider using a name that reflects its purpose, such as 'styleObject' or 'cssProperties'.